analyze_issue_metadata.py

```diff
@@ -0,0 +1,186 @@
+import csv
+import json
+import os
+import sys
+
+from datetime import datetime as dt
+
+import github3
+
+import get_issue_metadata as gim
+
+
+def main(state):
+
+    """
+    Command line tool that reads in an input file
+    containing GitHub repository names, calls read_issue_metadata
+    and returns a JSON file of issue data.
+
+    Usage (command line):
+        $ python analyze_issue_metadata.py googlecloudplatform repos.txt
+
+    Args:
+        state: GitHub state of issue ('open', 'closed'
+               or 'all')
+        sys.argv[1]: github owner or organization
+        sys.argv[2]: path to input file containing names of repos belonging to
+                     owner or org in sys.argv[1].  One repo name per line.
+
+    Returns:
+        A JSON of issue metadata
+
+    Raises:
+        N/A
+    """
+
+    org = str(sys.argv[1])
+
+    with open(sys.argv[2]) as f:
+        repos = f.readlines()
+    repos = [x.strip() for x in repos]
+
+    # Authenticate with GitHub using Personal Access Token
+    g = github3.login(token=os.environ['GH_TOKEN'])
+
+    data = gim.get_issue_metadata(g, org, repos, state)
+
+    # Dump the data to a JSON file
+    data_json = json.dumps(data)
+    file_time = dt.now().strftime("%Y%m%d")
+    print 'Dumping data to json file'
+    with open('../output_files/' + file_time + '_raw_veneer_issue_meta.json',
+              'w') as f:
+        data = json.dump(data, f, sort_keys=True, indent=4, separators=(
+             ',', ': '))
+
+    return data_json
+
+
+def analyze_issue_metadata(data):
+
+    p0_labels = ['Priority: P0', 'priority: P0', 'priority: p0', 'P0', 'p0']
+    p1_labels = ['Priority: P1', 'priority: P1', 'priority: p1', 'P1', 'p1']
+    p2_labels = ['Priority: P2+', 'priority: P2+', 'priority: p2+',
+                 'P2+', 'p2+', 'Priority: P2', 'priority: P2', 'priority: p2',
+                 'P2', 'p2']
+    fr_question = ['Type: Enhancement', 'Type: Question', 'type: enhancement',
+                   'type: question', 'enhancement']
+
+    analysis = {}
+
+    # Load/decode the data JSON
+    data_decode = json.loads(data)
+
+    for repo in data_decode:
+        if repo == 'updated':
+            continue
+        else:
```

Comment:
This `else` is unnecessary (since the `continue` statement effectively does the same). You can get rid of it and dedent everything one level.